### PR TITLE
Hooman pr

### PR DESCRIPTION
**Pull request by hooman96**
remove the unnecessary comments. 

add the features for migration

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -16,6 +16,5 @@
 # show_version: bool = __argv.version
 
 hello: str = f"hi{f', {username}' if username else ''}"
-# version: str = f" - python version : {sys.version_info.major}.{sys.version_info.minor}" if show_version else ""
```
Comment (hooman96) [inline main.py:19]: why are you printing the python version?

Comment (jooa32) [reply main.py:19]: It was just for testing. I'll remove for sure

File: service_migration.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    # Download the file from S3
+    s3_client.download_file(source_bucket, key, key)
+    print(f'Downloaded {key} from S3')
+
+    # Upload the file to GCS
+    blob = destination_bucket.blob(key)
+    blob.upload_from_filename(key)
+    print(f'Uploaded {key} to GCS')
+
+    # Clean up the local file
```
Comment (hooman96) [inline service_migration.py:24]: let's verify that data is at the destination before cleaning up

File: service_migration.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+def main():
+    # List objects in the source S3 bucket
+    objects = s3_client.list_objects_v2(Bucket=source_bucket)
+    if 'Contents' in objects:
+        for obj in objects['Contents']:
+            key = obj['Key']
+            try:
+                migrate_file(s3_client, source_bucket, destination_bucket, key)
+            except Exception as e:
+                print(f'Failed to migrate {key}: {e}')
```
Comment (hooman96) [inline service_migration.py:37]: should we add a re-try mechanism

File: main.py (hunk 1 above)
Comment (hooman96) [reply main.py:19]: is your code complete yet?

**Review by hooman96 — COMMENTED** [review]
Please let me know when you are planning to wrap up this project

**Pull request merged**